frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostSecondaryStatus.tsx

```diff
@@ -1,25 +1,36 @@
 import * as React from 'react';
 import { SecondaryStatus } from '@console/shared';
 import { BareMetalHostKind } from '../../types';
-import { getHostPowerStatus, getHostProvisioningState } from '../../selectors';
+import {
+  getHostPowerStatus,
+  getHostProvisioningState,
+  isHostScheduledForRestart,
+} from '../../selectors';
 import { HOST_POWER_STATUS_POWERED_ON, HOST_REGISTERING_STATES } from '../../constants';
 
 type BareMetalHostSecondaryStatusProps = {
   host: BareMetalHostKind;
 };
 
+export const HOST_SCHEDULED_FOR_RESTART = 'Scheduled for restart';
```

Comment:
When reading [Reboot API](https://github.com/metal3-io/metal3-docs/blob/master/design/reboot-interface.md), the semantics of the action is just "restart is requested", means it is "planed to happen when all conditions are met". There is no guarantee this to ever happen.

If/when the restart actually starts, the progress will be denoted by `Powering off` status.

I understand this does not conform the common sense for this action but it follows the declarative semantics implemented on the backend.

If the user needs "to force the restart now", there's still the `Power on|off` button.

Wdyt?

Reply:
Ah okay, I read the link more carefully this time. 🙂

How about we change "Scheduled for restart" to "Restart pending" then, which aligns a little more closely with the API's `pendingRebootSince` field?

I personally like the word "Restart", but I wonder if we should align even more closely to the API and call this "Reboot" everywhere in the UI instead. So "Reboot pending" and tweaks to the action label, modal text, etc.

WDYT?

Reply:
I think we do not need to be aligned on this with the backend since there are loose constraints for naming there.

The `openshift/console` uses `restart`. The `Cluster incidents` in ACM use `reststart` as well. So I am for keeping consistency on that level.

Reply:
I have changed "Scheduled for restart" to "Restart pending".

Reply:
Sticking with `Restart` sounds good, thanks Marek!